### PR TITLE
docs: add mqtt 5 beta banner and sponsors section

### PR DESCRIPTION
**Pull request by robertsLando**
Also did a quick lint fix on readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -834,15 +917,25 @@ See the [CONTRIBUTING.md](https://github.com/mqttjs/MQTT.js/blob/master/CONTRIBU
...
+| Matteo Collina     | [GitHub/mcollina](https://github.com/mcollina)     | [Twitter/@matteocollina](http://twitter.com/matteocollina) |
+| Maxime Agor        | [GitHub/4rzael](https://github.com/4rzael)         | [Twitter/@4rzael](http://twitter.com/4rzael)               |
+| Siarhei Buntsevich | [GitHub/scarry1992](https://github.com/scarry1992) |                                                            |
+| Daniel Lando    | [GitHub/robertsLando](https://github.com/robertsLando) |                                                           |
+
+<a name="sponsor"></a>
+
+## Sponsor
 
+If you want to sponsor MQTT.js, consider sponsoring his main author and active maintainers:
```
Comment (vishnureddy17) [inline README.md:932]:
```suggestion
If you would like to support MQTT.js, please consider sponsoring the author and active maintainers:
```

**Review by vishnureddy17 — APPROVED** [review]

**Comment by robertsLando** [comment]
cc @mcollina just to inform you, a sponsor section has been added to project README

**Review by vishnureddy17 — APPROVED** [review]

**Comment by robertsLando** [comment]
Tests are flaky, hopefully #1534 will fix this

**Comment by robertsLando** [comment]
@vishnureddy17 FYI I tried to make a new beta release using the workflow. It went well except for the last step where it should push the release and changelog to main branch: https://github.com/mqttjs/MQTT.js/actions/runs/5409237552/jobs/9829130699#step:7:1054

I will see if/how I can fix this

**Comment by robertsLando** [comment]
I saw the email, will give that a look tomorrow. Thanks!

**Pull request merged**